lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json

```diff
@@ -0,0 +1,354 @@
+{
+	"info": {
+		"_postman_id": "c2101915-caa3-4044-a6c4-e32fe21db062",
+		"name": "Lighty-RNC",
+		"description": "Example requests viable in RNC application.\nChange target of the request according to your deployment.",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Device managment",
+			"item": [
+				{
+					"name": "Connect NETCONF device",
+					"request": {
+						"auth": {
```

Comment:
This is the only request with basic-auth. Either add it to all requests or remove it.

Reply:
Done, removed it